src/handlers/api.js

```diff
@@ -124,6 +125,17 @@ export const apiGetAccountBalances = async (
   }
 };
 
+/**
+ * @desc get unique tokens
+ * @param  {String}   [address = '']
+ * @return {Promise}
+ */
+export const apiGetAccountUniqueTokens = async (address = '') => {
+  const data = await api.get(`/get_unique_tokens/${address}`);
+  const result = parseAccountUniqueTokens(data);
+  return result;
```

Comment:
you could just `return parseAccountUniqueTokens(data);` instead of assigning it to `result`

Reply:
yep